usr/lib/pkcs11/ep11_stdll/ep11_specific.c

```diff
@@ -2565,15 +2597,19 @@ static CK_RV import_DH_key(STDLL_TokData_t * tokdata, SESSION * sess,
             TRACE_INFO("%s wrapping unwrap key rc=0x%lx blob_size=0x%zx\n",
                        __func__, rc, *blob_size);
         }
+
+        cleanse_attribute(dh_key_obj->template, CKA_VALUE);
     }
 
 import_DH_key_end:
```

Comment:
check_key_attributes exit pint should be mapped to this label ?

Reply:
fixed